invenio_users_resources/records/mappings/os-v2/users/user-v2.0.0.json

```diff
@@ -1,4 +1,25 @@
 {
+  "settings": {
+    "index": {
+      "max_ngram_diff": 2
+    },
+    "analysis": {
+      "analyzer": {
+        "ngram_3_to_5_analyzer": {
+          "type": "custom",
+          "tokenizer": "ngram_tokenizer_3_to_5",
+          "filter": ["lowercase"]
+        }
+      },
+      "tokenizer": {
+        "ngram_tokenizer_3_to_5": {
```

Comment:
@jrcastro2 
I was looking into the [`edge_ngram` tokenizer](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-edgengram-tokenizer.html), it will create tokens with the first letter as the anchor. I might have less context, but I don't think one would search from the middle of the word. WDYT? CC: @carlinmack

Reply:
Good point! From what I read in the docs I only see one thing that might not be good for us:
> They are useful for querying languages that don’t use spaces or that have long compound words, like German.

In our use case we do use spaces, we have funders, authors ...  and sometime we want to search for terms that are after the space. If there is an n-gram tokenizer able to start for each word from the beginning I would agree that it's better.

Reply:
I couldn't find the bit you are referencing about German in the docs..

In this PR we are now using the edge_n_gram filter which tokenizes each word from the beginning :)